src/models/TriggerDefinition.js

```diff
@@ -0,0 +1,24 @@
+import { ComponentDefinition } from 'leto-modelizer-plugin-core';
+
+/**
+ * Trigger definition.
+ */
+class TriggerDefinition extends ComponentDefinition {
+  /**
+   * Default constructor.
+   * @param {Object} [props] - Object that contains all properties to set.
+   * @param {String} [props.action] - Action name call by the trigger.
```

Comment:
String or string ?

Reply:
we are waiting for the pr that add the jsdoc ;)